discovery-infra/test_infra/assisted_service_api.py

```diff
@@ -166,6 +166,12 @@ def get_infra_env_by_cluster_id(self, cluster_id: str) -> List[models.infra_env.
 
     @classmethod
     def _download(cls, response: HTTPResponse, file_path: str, verify_file_size=False) -> None:
+        warnings.warn(
```

Comment:
Instead of deprecating these, does it make sense to just change their implementation?

Reply:
I thought about it but we will anyway need to change the function signature (to pass infra_env or infra_env arguments).
Also, it's not really needed here, now we can download image directly from `infra_env`  using `infra_env.download_image`.
I would just delete it if no one was using it